src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/ProjectRazorJsonJsonConverter.cs

```diff
@@ -28,9 +28,9 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 return null;
             }
 
-            var (_, _, _, _, serializationFormat, filePath, configuration, rootNamespace, projectWorkspaceState, documents) = reader.ReadProperties(static (propertyName, arg) =>
+            var (_, _, _, _, serializationFormat, serializationFilePath, filePath, configuration, rootNamespace, projectWorkspaceState, documents) = reader.ReadProperties(static (propertyName, arg) =>
             {
-                var (reader, objectType, existingValue, serializer, serializationFormat, filePath, configuration, rootNamespace, projectWorkspaceState, documents) = (arg.reader, arg.objectType, arg.existingValue, arg.serializer, arg.serializationFormat, arg.filePath, arg.configuration, arg.rootNamespace, arg.projectWorkspaceState, arg.documents);
+                var (reader, objectType, existingValue, serializer, serializationFormat, serializationFilePath, filePath, configuration, rootNamespace, projectWorkspaceState, documents) = (arg.reader, arg.objectType, arg.existingValue, arg.serializer, arg.serializationFormat, arg.serializationFilePath, arg.filePath, arg.configuration, arg.rootNamespace, arg.projectWorkspaceState, arg.documents);
```

Comment:
Oh my, this is quite the parameter-fest. 10 items seems like an awful lot for a tuple.

Reply:
ya it's all in order to avoid the capturing lambda 😄